app/src/main/java/com/keylesspalace/tusky/util/TouchDelegateHelper.kt

```diff
@@ -12,25 +27,26 @@ import android.view.ViewGroup
  * Expands the touch area of the give row of views to fill the space in-between them, using a
  * [TouchDelegate].
  */
-fun ViewGroup.expandTouchSizeToFillRow(children: List<View>) {
+fun ViewGroup.expandTouchSizeToFillRow(children: List<View?>) {
     addOnLayoutChangeListener { _, _, _, _, _, _, _, _, _ ->
         touchDelegate = CompositeTouchDelegate(
             this,
-            children.mapIndexed { i, view ->
-                val rect = Rect()
-                view.getHitRect(rect)
-                val left = children.getOrNull(i - 1)
-                if (left != null) {
-                    // extend half-way to previous view
-                    rect.left -= (view.left - left.right) / 2
-                }
-                val right = children.getOrNull(i + 1)
-                if (right != null) {
-                    // extend half-way to next view
-                    rect.right += (right.left - view.right) / 2
+            children.filterNotNull()
```

Comment:
Worried this will mess up the before/after indexing, maybe filter before passing them in?

Reply:
yes, thx!